Dockerfile.release

```diff
@@ -1,23 +1,31 @@
-FROM bitwalker/alpine-erlang:19.2.1b
+FROM phusion/baseimage
```

Comment:
All right. Fair enough. It'd be good to know why exactly alpine has this issues.